### ISSUE TITLE
Nested bullet-lists lack important newlines

### ISSUE THREAD

**Issue by jmacdotorg** [keedi/Markdown-Pod#7]
The output of nested bullet-lists is missing a newline or two.

This:

```
* Fruits
    * Apples
    * Oranges
```

Becomes this:

```
=encoding utf8

=over

=item *

Fruits
=over

=item *

Apples


=item *

Oranges


=back



=back
```

Without an additional newline between e.g. "Fruits" and the following "=over", programs like Perldoc read that as a paragraph containing the content "Fruits =over".

**Comment by keedi** [comment]
Thanks for reporting. I'll check and fix it asap. :)

**Comment by olafhering** [comment]
@keedi : Did you have a chance to look at this?

**Comment by zmughal** [comment]
I have made a small fix for this at <https://github.com/keedi/Markdown-Pod/pull/10>.

**Comment by keedi** [comment]
Thx!! @zmughal++

@jmacdotorg, @olafhering:
I'm sorry I could not keep "ASAP". Although it was too late, I'm preparing to release. Please wait a little longer.

**Comment by keedi** [comment]
I've just released [0.007](https://metacpan.org/release/KEEDI/Markdown-Pod-0.007).

Thanks again. :)
